packages/boxed-expression-component/src/table/BeeTable/BeeTableHeader.tsx

```diff
@@ -229,6 +228,7 @@ export function BeeTableHeader<R extends object>({
                     <InlineEditableTextInput
                       setEditing={setEditing}
                       columnIndex={columnIndex}
+                      columnSpan={thRef.current?.colSpan ?? 1}
```

Comment:
I think this is 'rowSpan' because we are merging rows not columns

Reply:
this was removed, I simplified the PR, now there is no extension of existing selection API with `columnSpan`